chalicelib/app_utils.py

```diff
@@ -403,7 +408,7 @@ def view_foursight(environ, is_admin=False, domain="", context="/"):
     queue_attr = get_sqs_attributes(get_sqs_queue().url)
     running_checks = queue_attr.get('ApproximateNumberOfMessagesNotVisible')
     queued_checks = queue_attr.get('ApproximateNumberOfMessages')
-    first_env_favicon = get_favicon(servers[0]) # use first env
+    first_env_favicon = get_favicon(servers[0]) if len(servers) > 0 else get_favicon('ff')  # use first env, default FF
```

Comment:
Could you add a comment saying why "first" is relevant here? I'm out of context so don't know if both cgap and ff servers could be in play or whether it's just that you know all servers are cgap or all are ff.

Reply:
When navigating to foursight it is legal to navigate to http://foursight.4dnucleome.org/view/data,cgap for example at which point for favicon we just select the first one in the specified list.